<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyUser</class>
 <widget class="QWidget" name="modifyUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改用户信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(251,102,102, 200), stop:1 rgba(20,196,188, 210));</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 50px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QLineEdit" name="newPwd1">
        <property name="styleSheet">
         <string notr="true">font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;
height: 30px;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="notice">
        <property name="styleSheet">
         <string notr="true">height: 30px;
font: 25 14pt '微软雅黑 Light';
background-color: rgb(255, 255, 255);
color: rgb(255, 85, 127);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="newPwd2">
        <property name="styleSheet">
         <string notr="true">font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;
height: 30px;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="cancel">
        <property name="styleSheet">
         <string notr="true">QPushButton {
height: 30px;
font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;

}
QPushButton:hover {
    background-color: rgb(22,218,208);
}
QPushButton:pressed {
    background-color: rgb(17,171,164);
}
</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="oldPwd">
        <property name="styleSheet">
         <string notr="true">font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;
height: 30px;</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="save">
        <property name="styleSheet">
         <string notr="true">QPushButton {
height: 30px;
font: 25 14pt '微软雅黑 Light';
color: rgb(31,31,31);
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);
border-radius:15px;

}
QPushButton:hover {
    background-color: rgb(22,218,208);
}
QPushButton:pressed {
    background-color: rgb(17,171,164);
}
</string>
        </property>
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>newPwd1</zorder>
     <zorder>notice</zorder>
     <zorder>cancel</zorder>
     <zorder>oldPwd</zorder>
     <zorder>save</zorder>
     <zorder>newPwd2</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
